portfolio/src/main/webapp/scripts/server-dev.js

```diff
@@ -12,18 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
  * Fetches the previously entered comments from the server and inserts each
  * comment as a list item of the 'comments' <ul> element.
+ *
+ * An option to determine the maximum number of comments is also included
+ * using a Query String parameter created from the num-comments form. When
+ * the page is (re-)loaded, the number of comments displayed is determined
+ * from the selection in the previous session. Otherwise, the most recently
+ * submitted number selection will be used. The number of comments will
+ * also never exceed the number of total comments returned from the datastore.
  */
 function getCommentsThread() {
   fetch('/comment-data')
       .then(response => response.json())
       .then((commentList) => {
         const commentThread = document.getElementById('comments-thread');
-        commentList.forEach((comment) => {
-          commentThread.appendChild(createListElement(comment));
-        });
+        const urlParams = new URLSearchParams(window.location.search);
+
+        // Determine the number of comments to display.
+        let numComments = urlParams.get('num-comments');
+        const numCommentsStored = parseInt(
+            sessionStorage.getItem('numComments'));
```

Comment:
fit one line?

Reply:
It is 83 characters if I make it one line so I wanted to split it up per the style guide. Do you think I should keep it as is or put it on one line?